src/Nordea/Components/MultiSelectDropdown.elm

```diff
@@ -194,17 +205,114 @@ view attrs dropdown =
                         display none
                     , flexDirection column
                     , backgroundColor Colors.white
-                    , border3 (rem 0.0625) solid Colors.mediumGray
+                    , border3
+                        (if List.isNotEmpty selectItems then
+                            rem 0.0625
+
+                         else
+                            rem 0
+                        )
+                        solid
+                        Colors.mediumGray
                     , Themes.borderColor Colors.nordeaBlue
                     , borderBottomLeftRadius (rem 0.25)
                     , borderBottomRightRadius (rem 0.25)
                     , boxSizing borderBox
                     ]
                 ]
-                (List.indexedMap viewSelectItems dropdown.optionGroups)
+                selectItems
```

Comment:
Looks like `selectItems` is a list of `ul` tags. I dont think ul tags should be direct children of ul`s, but wrapped in li`s.

Reply:
I see it is not introduced by your changes, but good to get fixed

Reply:
done